plugins/kafka-backend/config.d.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export interface Config {
+  kafka?: {
+    clientId: string;
+    brokers: string[];
+    ssl?: {
```

Comment:
This probably also needs to support a bool for SASL support. I guess that can be added after along with a password / client secret.

Reply:
Yeah as I said on Discord I'm finding myself adding more and more to this and it's big enough so it's best to add it separately. Anyway it's just configs I can pass directly to Kafka.js

Reply:
Yep, just pointing out :)